templates/downloads/index.html

```diff
@@ -38,6 +39,8 @@ <h1 class="call-to-action">Download the latest version of Python</h1>
 {% block content %}
                 <div class="row download-list-widget">
 
+                    {% render_active_banner False %}
```

Comment:
Nit: Passing `psf_pages_only=False` would increase readability here.

Reply:
again, will be addressed with excellent suggestion from jaap3